src/amo/components/Collection/index.js

```diff
@@ -500,7 +578,9 @@ export const mapStateToProps = (
   const filtersFromLocation = convertQueryParamsToFilters(location.query);
   const filters = {
     page: filtersFromLocation.page || 1,
-    sort: filtersFromLocation.sort || COLLECTION_SORT_DATE_ADDED_DESCENDING,
+    collectionSort:
```

Comment:
So I see why you've changed the `sort` parameter but the problem is that it remains in the URL when navigating to the search results page. I think we should handle this differently.